packages/@expo/cli/src/utils/env.ts

```diff
@@ -141,6 +141,11 @@ class Env {
   get EXPO_USE_PATH_ALIASES(): boolean {
     return boolish('EXPO_USE_PATH_ALIASES', false);
   }
+
+  /** **Experimental:** Use the network inspector by overriding the metro inspector proxy with a custom version */
+  get EXPO_USE_CUSTOM_INSPECTOR_PROXY(): boolean {
```

Comment:
Add to the expo-cli docs https://docs.expo.dev/workflow/expo-cli/#environment-variables

Reply:
Done